[2023-12-13T23:36:50.773+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: my_dag.create_domains_info_table scheduled__2023-12-12T00:00:00+00:00 [queued]>
[2023-12-13T23:36:50.910+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: my_dag.create_domains_info_table scheduled__2023-12-12T00:00:00+00:00 [queued]>
[2023-12-13T23:36:50.919+0000] {taskinstance.py:1361} INFO - Starting attempt 2 of 2
[2023-12-13T23:36:52.099+0000] {taskinstance.py:1382} INFO - Executing <Task(PostgresOperator): create_domains_info_table> on 2023-12-12 00:00:00+00:00
[2023-12-13T23:36:52.176+0000] {standard_task_runner.py:57} INFO - Started process 582 to run task
[2023-12-13T23:36:52.329+0000] {standard_task_runner.py:84} INFO - Running: ['***', 'tasks', 'run', 'my_dag', 'create_domains_info_table', 'scheduled__2023-12-12T00:00:00+00:00', '--job-id', '81', '--raw', '--subdir', 'DAGS_FOLDER/my_dag.py', '--cfg-path', '/tmp/tmpebvgszer']
[2023-12-13T23:36:52.339+0000] {standard_task_runner.py:85} INFO - Job 81: Subtask create_domains_info_table
[2023-12-13T23:36:52.484+0000] {task_command.py:416} INFO - Running <TaskInstance: my_dag.create_domains_info_table scheduled__2023-12-12T00:00:00+00:00 [running]> on host dde03b621bd8
[2023-12-13T23:36:52.644+0000] {taskinstance.py:1662} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='my_dag' AIRFLOW_CTX_TASK_ID='create_domains_info_table' AIRFLOW_CTX_EXECUTION_DATE='2023-12-12T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='2' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2023-12-12T00:00:00+00:00'
[2023-12-13T23:36:52.646+0000] {sql.py:274} INFO - Executing: 
        CREATE TABLE IF NOT EXISTS DomainsInfo (
            id  SERIAL PRIMARY KEY,
            domain VARCHAR(128),
            info TEXT
        );
        
[2023-12-13T23:36:52.666+0000] {base.py:73} INFO - Using connection ID 'postgres_hw2' for task execution.
[2023-12-13T23:36:52.699+0000] {base.py:73} INFO - Using connection ID 'postgres_hw2' for task execution.
[2023-12-13T23:36:52.701+0000] {taskinstance.py:1937} ERROR - Task failed with exception
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/providers/common/sql/operators/sql.py", line 280, in execute
    output = hook.run(
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/providers/common/sql/hooks/sql.py", line 385, in run
    with closing(self.get_conn()) as conn:
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/providers/postgres/hooks/postgres.py", line 155, in get_conn
    self.conn = psycopg2.connect(**conn_args)
  File "/home/airflow/.local/lib/python3.8/site-packages/psycopg2/__init__.py", line 122, in connect
    conn = _connect(dsn, connection_factory=connection_factory, **kwasync)
psycopg2.OperationalError: connection to server at "localhost" (127.0.0.1), port 5433 failed: Connection refused
	Is the server running on that host and accepting TCP/IP connections?
connection to server at "localhost" (::1), port 5433 failed: Cannot assign requested address
	Is the server running on that host and accepting TCP/IP connections?

[2023-12-13T23:36:52.716+0000] {taskinstance.py:1400} INFO - Marking task as FAILED. dag_id=my_dag, task_id=create_domains_info_table, execution_date=20231212T000000, start_date=20231213T233650, end_date=20231213T233652
[2023-12-13T23:36:52.761+0000] {standard_task_runner.py:104} ERROR - Failed to execute job 81 for task create_domains_info_table (connection to server at "localhost" (127.0.0.1), port 5433 failed: Connection refused
	Is the server running on that host and accepting TCP/IP connections?
connection to server at "localhost" (::1), port 5433 failed: Cannot assign requested address
	Is the server running on that host and accepting TCP/IP connections?
; 582)
[2023-12-13T23:36:52.885+0000] {local_task_job_runner.py:228} INFO - Task exited with return code 1
[2023-12-13T23:36:52.980+0000] {taskinstance.py:2778} INFO - 0 downstream tasks scheduled from follow-on schedule check
